nservicebus/learning-transport/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Learning Transport
+component: LearningTransport
+reviewed: 2017-05-01
+---
+
+The Learning Transport stores all message actions in the local file system relative to the current solution directory.
```

Comment:
> The Learning Transport simulates queuing infrastructure by storing all message actions in the local file system.

I think opening with "simulates" or something similar helps to hammer home the non-production nature of it.